QA.md

```diff
@@ -0,0 +1,130 @@
+# Test Sequence
+
+### Those tests must be done in an Electron application. Some features might work on the web browser but show different behavior when using electron.
+
+- **Initialization**
+1. [] Try to click 'Get started' without selecting checkbox (must show error).
+1. [] Start new wallet.
+1. [] Try to click 'Create my words' without selecting checkbox (must show error).
+1. [] Select to do backup now. (Really save those words, we will load this wallet later with them).
+1. [] Click 'Validate' before selecting all the words.
+1. [] Click 'Validate' with all words selected but in the wrong order.
+1. [] Validate with correct order.
+1. [] Try to select insecure password (like '123'). It must show error.
+1. [] Select password and PIN.
+
+- **New token error**
+1. [] Click on 'Custom tokens', then 'Create a new token'.
+1. [] Type 'Test token' in the token name, 'Test' in the symbol and 100 in the amount.
+1. [] Validates that the deposit required is 1.00 HTR.
+1. [] Try to create the token and receive an error that you don't have enough HTR.
+
+- **Wallet screen**
+1. [] It shows a red label with the testnet name in the top right corner of the screen.
+1. [] Copy the address and send HTR to it.
```

Comment:
Before this step, the user needs some balance. It's important that we make it clear, as in the last section we expected him to have none (when creating a token).

Reply:
Actually this is the step where it receives the first transaction. I don't see why the user needs balance before it.

> Copy the address and send HTR to it.

Reply:
Ahhh, I think I understand now the confusion. I need to make clear you have to send this HTR here from another wallet, isn't it? Will add this

Reply:
Yes. I thought the tester would be sending tokens to himself.

Reply:
Done, thanks